pkg/common/load/load.go

```diff
@@ -70,17 +70,24 @@ func Configs(configs []string, customConfig string, secretLocations []string) er
 		for _, folder := range secretLocations {
 			passthruSecrets := viper.GetStringMapString(config.NonOSDe2eSecrets)
 			if !strings.Contains(folder, "osde2e-credentials") && !strings.Contains(folder, "osde2e-common") {
-				continue
+				if viper.Get(config.Addons.IDs) != nil {
+					passthruSecrets["ocm-token-refresh"] = viper.Get("ocm.token").(string)
```

Comment:
You should be do `viper.GetString("ocm.token")` rather than casting `Get`'s output to a string.